dev/buildtool/spinnaker_commands.py

```diff
@@ -81,7 +67,7 @@ def init_argparser(self, parser, defaults):
             "min_halyard_version",
             defaults,
             None,
-            help="The minimum halyard version required.",
+            help="The minimum halyard version required if changed.",
```

Comment:
What does "if changed" mean?

Reply:
Thanks. This doesn't make sense here. Have reworded in: https://github.com/spinnaker/buildtool/pull/196/commits/996e11f8a726d99bb37bca5ed71be36f1147ba7c

Ideally it wouldn't be a required parameter so we can avoid supplying it if no need to bump. Still, only 2 params required at the moment so it's not the worst.